elementary/monitor/cli.py

```diff
@@ -46,6 +46,12 @@ def common_options(func):
         default=7,
         help="Set a limit to how far back should edr collect data.",
     )(func)
+    func = click.option(
+        "--env",
+        type=str,
+        default="dev",
+        help="The environment that you're running against. Usually this would be either 'dev' or 'prod'.",
```

Comment:
cosmetics - This flag indicates if you are running Elementary in prod or dev environment and will be reflected accordingly in the report. Value should be either 'dev' or 'prod' (default is 'dev').